src/test/perl/NCM/Component/brokennew.pm

```diff
@@ -0,0 +1,12 @@
+package NCM::Component::brokennew;
+
+use strict;
+use warnings;
+use LC::Exception;
+
+# Needs to hav a new method, ideally via NCM::Component inheritance
```

Comment:
So this test doesn't check anything yet? Could this be made clearer in the comments so that no one thinks we are checking for something we are not?